tests/.gitignore

```diff
@@ -0,0 +1 @@
+/*/core
```

Comment:
Maybe `core` and `core.*`

Reply:
Done. But I white-listed `core.c`, `core.h`, `core.txt`, and `core.md` again to avoid surprises.